frontend/src/interfaces/project.ts

```diff
@@ -7,13 +7,16 @@ export interface IProjectCard {
     name: string;
     id: string;
     createdAt: string;
-    health: number;
```

Comment:
what has changes that those are now optional? was it wrong before?

Reply:
Now it supports also archive, and it's easier to solve it this way then with generic.